### PR TITLE
Inline video fixes

### PR DESCRIPTION
**Pull request by benghaziboy**
@collin

### CODE REVIEW

**Comment by benghaziboy** [comment]
@jmelvnsn while we're here

File: elements/bulbs-video/bulbs-video.scss [hunk 1]
```diff
@@ -78,9 +78,18 @@ bulbs-video {
   .bulbs-video-poster-overlay {
     bottom: 0;
     left: 0;
-    position: absolute;
     right: 0;
     top: 0;
+
+    position: absolute;
```
Comment (jmelvnsn) [inline elements/bulbs-video/bulbs-video.scss]: Small but please move this rule back to alphabetical order.

**Comment by collin** [comment]
👍

**Comment by jmelvnsn** [comment]
Other than the small CSS change LG(OAT)TM

**Pull request merged**